validator/client/service.go

```diff
@@ -237,7 +239,7 @@ func (v *ValidatorService) Start() {
 	}
 
 	v.validator = valStruct
-	go run(v.ctx, v.validator)
+	go run(v.ctx, v.validator, hosts)
```

Comment:
if there are get host and update host functions on the validator, what about just adding hosts to the validator client? instead of passing it down as arguments?

Reply:
Ah, this makes a lot of sense. Instead of updating the host in the goroutine, we can have an `UpdateHost()` function without arguments that updates the host internally.